test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -24,6 +24,7 @@
 using Microsoft.SqlTools.ServiceLayer.Connection.Contracts;
 using Moq;
 using NUnit.Framework;
+using Microsoft.SqlServer.Management.SqlParser.Parser;
```

Comment:
This should be in order with the rest (general rule is System.* first, then everything else alphabetically)